src/templates/ar/2019/base.html

```diff
@@ -1,226 +1,97 @@
-{% extends "base.html" %}
+{% extends "base.html" {% extends "base/2019/base.html" %}
```

Comment:
```suggestion
{% extends "base/2019/base.html" %}
```

Reply:
Thanks! Typo. Was fine in other languages but this one slipped the net cause I couldn't test it.